owl/SOMA-ACT.owl

```diff
@@ -1,1088 +1,2462 @@
-Prefix(:=<http://www.ease-crc.org/ont/SOMA-ACT.owl#>)
```

Comment:
Can you save this as OWL Functional syntax as opposed to rdf/xml? Might reduce the diff size I must look at.

Reply:
Please save this as OWL functional syntax I am not going to look through a 2K size diff that's just there because of format issues.